### PR TITLE
alter css and html for portfolio page so text is inline with sidebar

### PR DESCRIPTION
**Pull request by loganmeetsworld**
[CircleCI](https://circleci.com/gh/18F/portfolios/tree/BRANCH_NAME)

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/18f/portfolios/BRANCH_NAME/)

## Changes proposed in this pull request

This changes some of the portfolio html and css to be more in line with what's in the API template. The image wasn't showing up and the text was a little confusing where it was positioned on the page. This changes the Alaska project page post from: 

Screen Shot 2021-06-22 at 3 20 04 PM

to this on desktop/mobile: 
Screen Shot 2021-06-22 at 3 19 26 PM
Screen Shot 2021-06-22 at 3 19 43 PM

## Notes

- One thing I noticed is that if an image isn't present it pulls the hero text to the center. I wasn't sure if this is something we wanted so I didn't touch it.
- I'll take a look at the project template (that the Vermont post inherits from) to see if we can have them all be consistent. 
- Finally, CSS is not my expertise so if I did something wrong (like adding the `inline` statement in the wrong place) let me know!

### CODE REVIEW

File: _includes/navigation/footer.html [hunk 1]
```diff
@@ -11,6 +11,9 @@ <h3 class="usa-footer__primary-link">Portfolios</h3>
               <li>
                <a class="usa-footer__secondary-link" href="{{ site.baseurl }}/national-security-intelligence/">National Security & Intelligence</a>
               </li>
+              <li>
+                <a class="usa-footer__secondary-link" href="{{ site.baseurl }}/state-local/">State & Local</a>
+               </li>
```
Comment (loganmeetsworld) [inline _includes/navigation/footer.html:16]: Little extra thing - just added State & Local to the footer!

**Comment by stvnrlly** [comment]
Oh nice! If you're going down this road, I suggest looking at [/_layouts/apis.html ](https://github.com/18F/portfolios/blob/main/_layouts/apis.html), which might either simplify things or complicate things

**Comment by loganmeetsworld** [comment]
@stvnrlly what do you mean "looking at"? Like, maybe just making everything use one layout instead of having apis/project/project_portfolios, etc.?

**Comment by loganmeetsworld** [comment]
@stvnrlly I went through and cleaned this up a bit, removing layouts with duplication or that weren't used anywhere. It looks really good on the two project pages we link to, but I also tried it on a bunch of the `no-link` pages too. 

Screen Shot 2021-06-23 at 9 09 26 AM
Screen Shot 2021-06-23 at 9 09 36 AM

**Review by stvnrlly — APPROVED** [review]

**Pull request merged**